libs/dataviews-client/src/merge-dataviews.ts

```diff
@@ -261,16 +261,21 @@ export function getDataviewsMerged(
     )
 
   const singleHeatmapDataview = [...activityDataviews, ...detectionDataviews].length === 1
+  const heatmapAnimatedMode = params.bivariateDataviews
+    ? HeatmapAnimatedMode.Bivariate
+    : HeatmapAnimatedMode.Compare
   // If activity heatmap animated generators found, merge them into one generator with multiple sublayers
   const mergedActivityDataview = activityDataviews?.length
     ? getMergedHeatmapAnimatedDataviews(activityDataviews, {
         ...params,
+        heatmapAnimatedMode,
```

Comment:
I have some doubts about how to integrate this because the bivariate mode is not compatible between these two layers. let's review it together tomorrow